prover-service/src/prover_config.rs

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Aptos Foundation
+
+use crate::groth16_vk::{OnChainGroth16VerificationKey, SnarkJsGroth16VerificationKey};
+use crate::utils;
+use aptos_keyless_common::input_processing::config::CircuitConfig;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+// Constants for the prover service configuration file
+const CIRCUIT_CONFIG_YML_FILE_NAME: &str = "circuit_config.yml";
+const GENERATE_WITNESS_JS_FILE_NAME: &str = "generate_witness.js";
+const MAIN_WASM_FILE_NAME: &str = "main.wasm";
+
+/// A simple struct representing an OIDC provider
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(deny_unknown_fields)]
+pub struct OidcProvider {
+    pub iss: String,
+    pub endpoint_url: String,
+}
+
+/// The prover service configuration
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(deny_unknown_fields)]
+pub struct ProverServiceConfig {
+    pub setup_dir: String,
+    pub resources_dir: String, // Directory with prover/verification key and witness gen binary
+    pub zkey_filename: String,
+    pub test_verification_key_filename: String,
```

Comment:
Anyone know why this is called `test_verification_key`, but we seem to use it in prod? 🤔 (See below).

Reply:
it was initially just for debugging/observability. later it was used by critical logic but we didn't bother renaming it everywhere...

(and now it's against only a observability thing)